### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by KnorpelSenf**

### CODE REVIEW

File: lib/TasksResource.ts [hunk 1]
```diff
@@ -180,49 +223,50 @@ export interface TaskCaptureData {
...
     engine_version?: string;
-    filename?: string;
+    timeout?: number;
 
     [option: string]: any;
 }
 
-export interface TaskMetadataData {
+interface TaskMetadataWrite {
+    operation: 'metadata/write';
```
Comment (KnorpelSenf) [inline lib/TasksResource.ts:259]: Is this the correct operation type? I'm not 100 % from the docs.

**Review by KnorpelSenf — COMMENTED** [review]
@josiasmontag I think this should be good for review. It would be cool if you could quickly go through the changelog to make sure I did not miss anything.

Otherwise, I only have one thing where I'm not completely sure:

File: package.json [hunk 2]
```diff
@@ -16,25 +16,25 @@
         "url": "https://github.com/cloudconvert/cloudconvert-node/issues"
     },
     "dependencies": {
-        "axios": "^0.21.1",
+        "axios": "^0.26.1",
         "esm": "^3.2.25",
```
Comment (KnorpelSenf) [inline package.json:20]: This should be a dev dep, no?

Comment (josiasmontag) [reply package.json:20]: Yes, should be dev dep

File: lib/TasksResource.ts (hunk 1 above)
Comment (josiasmontag) [reply lib/TasksResource.ts:259]: Yes it is correct

**Comment by josiasmontag** [comment]
Looks fine!
Could you let me know your email address? I have a little offer for you 😊

File: lib/TasksResource.ts (hunk 1 above)
Comment (KnorpelSenf) [reply lib/TasksResource.ts:259]: Perfect!

File: package.json (hunk 2 above)
Comment (KnorpelSenf) [reply package.json:20]: Alright, I'll update it in a few hours when I have time, and then it's ready to be merged

**Comment by KnorpelSenf** [comment]
> Looks fine!

Nice!

> Could you let me know your email address? I have a little offer for you 😊

Sure, you can just send it to the email address that I'm using to author my commits (e.g. visible [here](https://github.com/cloudconvert/cloudconvert-node/commit/7ed3165fc5395d67cc7d5b1c3964d867cb0dbac8.patch) )

**Pull request merged**